### PR TITLE
Hello feature

### PR DESCRIPTION
**Pull request by medijdir**
hi, 🤝
In this project, I have used:
→  Html Technologies ⌨️ 
→  CSS 💻
→  Linters configure 🎲
→ GitHub flow  🪄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+
+👤 **Author**
+
+- GitHub: [@medijdir](https://github.com/medijdir)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/jdir-mehdi-a3055a256/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
```
Comment (Olamarx) [inline README.md:104]: You probably forgot to add the future features, this is required too.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+
+- **[A header with a purple text "Hello Microverse Project"]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]
```
Comment (Olamarx) [inline README.md]: - [ ] Since there is no live demo, you should remove this section and if you have deployed it, you can add it.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @medijdir, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Readme file professional ✔️
- Branch name meaningful ✔️
- Commit messages meaningful ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] From your linters, I can see that you tried to deploy the app but not successful and as a result the linter fails, while deployment is not required, I will encourage you to fix it so that your linters would all be checked.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — APPROVED** [review]
Hi @medijdir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters setup done correctly
Ignore node_modules folder and test file

## Optional suggestions
_Great done by applying changes requested by the previous reviewer , I would just suggest that you check your readme preview while making changes to be sure about the output as the elements are not well aligned this is important to make your work clean and professional.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medijdir** [comment]
hi @souad988 and @Olamarx, 🤝🤝
Thanks ✌️✌️for helping me to approve 🎉🎈🎇🪄this challenge, I feel very happy  .
TennesseeVolsGIF

**Pull request merged**